QMChatService/QMChatService/Categories/QBChatMessage+QMCustomParameters.m

```diff
@@ -330,4 +330,9 @@ - (BOOL)isMediaMessage {
 	return self.attachments.count > 0 || self.attachmentStatus == QMMessageAttachmentStatusLoading;
 }
 
+- (BOOL)isStickerMessage {
```

Comment:
[isNotificationMessage](https://github.com/908Inc/q-municate-services-ios/blob/development/QMChatService/QMChatService/Categories/QBChatMessage%2BQMCustomParameters.m#L323) getter should be updated now. Sticker is not notification.